data/rulei/rulei.txt

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Rulei"
+		personality heroic vindictive staying mute
+		system "Kanguwa"
+		ship "vyu-Ira" "Solemnity"
+
+mission "Ira Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Kanguwa
+		fail
+		log "Factions" "Rulei" `The Rulei were likely an ancient technologically advanced civilization from long ago.`
+			`They communicate through some sort of invasive psychonic ability.`
+		conversation
+			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. The best way to describe it is an overwhelming sense of emptiness, but at the same time, the intensity of it makes it feels like something must be there. You barely have enough time to try and make sense of this oddity as only seconds later, you start to hear ominous whispering, as if something is behind you.`
+			choice
+				`	(Turn around.)`
+			`	You turn your head around to look, but every time, the source of the whispers remains at the back of your head, like an itch that can't be reached. You can't even remotely understand the language, and the whispers progressively increase in volume. The louder they get, the more there are.`
+			
+			`It sounds like thousands of varingly pitched voices are speaking at all at once now, and you senses all feel overloaded. Just before you think you can't take anymore, everything falls completely silent.`
+			scene "scene/rulei flash"
```

Comment:
From what I experienced it is best to have a choice node before a scene, it just looks better

Reply:
If you can think of something to put there, sure, but I can't think of anything that would fit without being weird.